packages/vite/src/node/plugins/css.ts

```diff
@@ -261,7 +261,8 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
                         await fileToUrl(file, config, this),
                         (config.server?.origin ?? '') + devBase
                       ),
-                      ssr
+                      ssr,
+                      false
```

Comment:
**refactor:** The default value of `isSelfAccepting` is now undefined, so we need to pass an explicit `false` value wherever nothing was previously passed.